flask_oidc/__init__.py

```diff
@@ -119,32 +122,45 @@ def __init__(self, app=None, credentials_store=None, http=None, time=None,
         # By default, we do not have a custom callback
         self._custom_callback = None
 
+        # In the beginning we know nothing ...
+        self.client_secrets = None
+
         # get stuff from the app's config, which may override stuff set above
-        if app is not None:
+        
+        if app:
```

Comment:
Why did you change this line? The intention is that it'll be called with either a `flask.Flask` instance or `None`.